test/schema/smbgTest.js

```diff
@@ -43,5 +43,16 @@ describe('schema/cbg.js', function(){
     helper.expectUnitConversion(goodObject, 'value');
   });
 
+  describe('subType', function(){
+    var withSubType = _.assign({}, goodObject, {subType: ''});
+    helper.okIfAbsent(goodObject, 'subType');
+    helper.expectStringField(withSubType, 'subType');
```

Comment:
am i missing something? isn't this just testing twice that it will pass muster if the subType = ''